server/embed/etcd.go

```diff
@@ -81,10 +81,16 @@ type Etcd struct {
 
 	cfg   Config
 	stopc chan struct{}
```

Comment:
Can you add a similar explanation to stopc, closeOnce and sctxs[].serversC?

Reply:
Done.